Scripts/StixParser/StixParser.py

```diff
@@ -15,12 +15,29 @@
     "url:": "URL",
     "domain-name:": "Domain",
     "email:": "Email",
-    "email-message": "Email"
+    "email-message": "Email",
+    "win-registry-key:key": "Registry Path Reputation",
```

Comment:
why do we need both ?
why isnt CVE here?

Reply:
because CVE entry has no `pattern` key, so it's parsed differently